.eslintrc.json

```diff
@@ -14,6 +14,7 @@
     "plugin:prettier/recommended"
   ],  
   "rules": {
+    "no-unused-vars": "off",
```

Comment:
This should have been disabled already since it conflicts with `@typescript-eslint/no-unused-vars` which replaces this rule for TS.